lang/go/bindings/examples/distributor/pkg/distributor/cmd.go

```diff
@@ -0,0 +1,115 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2022-2023 Intel Corporation
+ */
+
+package distributor
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/CloudNativeDataPlane/cndp/lang/go/bindings/cne"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	version    = "v0.0.1"
```

Comment:
The version string is major.minor.patch, this means it should be 0.1.0 instead.

Reply:
Updated